Dockerfile

```diff
@@ -1,14 +1,10 @@
-# Docker image for the Drone Webhook plugin
-#
-#     cd $GOPATH/src/github.com/drone-plugins/drone-webhook
-#     make deps build docker
-
 FROM alpine:3.3
```

Comment:
You should use the same format as s3-sync or s3-cache, no need for alpine, that way we can directly create multiple architectures without native agents.

Reply:
I'll copy the dockerfiles from the s3-cache, but I'm not quite seeing what the difference is.  is it the FROM alpine:3.3 as alpine that allows that?

Reply:
s3-cache copies ca-certificates.crt from alpine but builds only a scratch image with a multi stage build, that's why it also works for different architectures